app/pages/sections/additional-services/select/routes.js

```diff
@@ -108,8 +112,15 @@ export const additionalServicesSelectRoutes = (authProvider, addContext, session
 
   router.get('/additional-service/price/recipient', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
     const { orderId } = req.params;
+    const solutionId = sessionManager.getFromSession({ req, key: 'selectedSolutionId' });
+    const solutionData = await getSolution({ solutionId });
```

Comment:
I still don't think getSolution is what we should be calling here. @sam-roberts-bjss is this right? will this endpoint also give us the additionalService data from BAPI?

I think we should be calling the `getCatalogueItem` instead.

Reply:
Yeah that's right. I think here we should really be using session to our advantage & retrieve the name of the additional service from there. If its not been stored in session we should probably add that.